src/html/index.html

```diff
@@ -47,20 +47,20 @@
   </canvas>
 
   <div class="cookies hidden">
-    <p class="cookies__text">This website uses cookies to ensure you get the best experience on our website.</p></li>
-    <button class="btn btn--cookies cookies__accept">Got it!</button>
+    <p class="cookies__text" alt="This website uses cookies to ensure you get the best experience on our website.">This website uses cookies to ensure you get the best experience on our website.</p></li>
```

Comment:
Alt tags are only useful on tags which have no value or inner text which is explanatory. That is, a button is fine, or a image, but a <p> tag don't need it.

Reply:
ah yes! of course :)